### DISCUSSION TITLE
.azw3 File extension

### DISCUSSION THREAD

**Discussion by Zeyadas** [koreader/koreader#7771] [Ideas]
I'm aware that the .azw3 is a kindle file extension.
I downloaded a file/book in that extension and koreader couldn't open it..
Are you going to make koreader support that file extension?

**Comment by mergen3107** [comment]
Most probably not: https://github.com/koreader/koreader/issues/5845

So you might want to convert to epub

**Reply by Zeyadas** [reply]
OK. Thanks.
